src/components/TodoItem.tsx

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+// import { ErrorMessage } from '../App';
+import { ErrorMessage } from './Errors';
+
+interface TodoItemProps {
+  id: number;
+  title: string;
+  completed: boolean;
+  isLoading: boolean;
+  onToggle: (id: number) => void;
+  onDelete: (id: number) => void;
+  onError: (error: string) => void;
+}
+
+export const TodoItem: React.FC<TodoItemProps> = ({
+  id,
+  title,
+  completed,
+  isLoading,
+  onToggle,
+  onDelete,
+  onError,
+}) => {
```

Comment:
Consider it will be better, it will be more readable. Also, you can move interface `TodoItem` to the separate file
```suggestion
interface TodoItem {
  id: number;
  title: string;
  completed: boolean;
  isLoading: boolean;
  onToggle: (id: number) => void;
  onDelete: (id: number) => void;
  onError: (error: string) => void;
};

interface Props {
  todo: TodoItem;
};

export const TodoItem: React.FC<Props> = ({ todo }) => {
```